content/en/docs/configuring-sdk/auth.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Configuring Authentication"
+linkTitle: "Authentication"
+date: "2023-12-01"
+description: Customizing service client authentication.
+---
+
+The {{% alias sdk-go %}} provides the ability to configure a custom
+auth scheme resolver to be used for a service. In most cases, the default
+configuration will suffice, but configuring custom authentication allows for
+additional behavior such as working with pre-release service features.
+
+## Auth scheme
+
+An auth scheme is defined as the workflow through which the SDK supplements an
+operation request with a caller identity.
+
+An auth scheme uses the following components:
+* A unique ID which identifies the scheme
+* An identity resolver, which returns a caller identity used in the signing process (e.g. your AWS credentials)
+* A signer, which performs the actual injection of caller identity into the operation's transport request (e.g. the `Authorization` HTTP header)
+
+The following auth schemes are used within the SDK:
+
+| Name | Scheme ID | Identity resolver | Signer | Notes |
+| ---  | --------- | ----------------- | ------ | ----- |
+| [SigV4](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_aws-signing.html) | `aws.auth#sigv4` | [aws.CredentialsProvider](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/aws#Credentials) | [v4.HTTPSigner](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/aws/signer/v4#Signer) | The current default for most AWS service operations. |
+| SigV4A | `aws.auth#sigv4a` | aws.CredentialsProvider | n/a | SigV4A usage is limited at this time, the signer implementation is internal. |
+| SigV4Express | `com.amazonaws.s3#sigv4express` | [s3.ExpressCredentialsProvider](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/s3#ExpressCredentialsProvider) | v4.HTTPSigner | Used for [Express One Zone](https://aws.amazon.com/s3/storage-classes/express-one-zone/) requests. |
+| HTTP Bearer | `smithy.api#httpBearerAuth` | [smithybearer.TokenProvider](https://pkg.go.dev/github.com/aws/smithy-go/auth/bearer#TokenProvider) | [smithybearer.Signer](https://pkg.go.dev/github.com/aws/smithy-go/auth/bearer#Signer) | Used by the codecatalyst service. |
+| Anonymous | `smithy.api#noAuth` | n/a | n/a | No authentication - no identity is required, and the request is not signed or authenticated. |
+
+In {{% alias sdk-go %}}, the identity and signer components of an auth scheme
+are configured on a service's client Options. The SDK will automatically pick
```

Comment:
nit: `Options` missing code formatting. throughout this page, some have it and some dont. we should prob make all uses of Options have it